pkg/vulnsrc/ubuntu/ubuntu.go

```diff
@@ -170,8 +170,12 @@ func defaultPut(dbc db.Operation, tx *bolt.Tx, advisory interface{}) error {
 			}
 
 			adv := types.Advisory{}
-			if status.Status == "released" {
+			status = StatusFromUbuntuStatus(status.Status)
```

Comment:
resolve type mismatch

Reply:
Thanks for spotting this.
type conflict resolved with var rename.

status was a conflicting var name.